lib/src/components/Banner.tsx

```diff
@@ -144,15 +110,67 @@ function	Banner({
 				}
 				{variant === 'split' ? <SplitVariant image={image as string} /> : null}
 				{variant === 'background' ? <BackgroundVariant image={image as string} /> : null}
-
-				{canClose ? (
-					<button onClick={onTryToClose} className={'absolute top-4 right-4'}>
-						<IconCross className={`w-6 h-6 cursor-pointer ${variant === 'split' && 'text-surface'}`} />
-					</button>
-				) : null}
 			</div>
 		</div>
 	);
 }
 
-export {Banner};
\ No newline at end of file
+
+export type	TBannerPagination = {
+	children: ReactElement[],
+	canClose?: boolean,
+	onClose?: () => void
+}
+
+function	BannerControlable({children, onClose, canClose = true}: TBannerPagination): ReactElement {
+	const	[currentSlide, set_currentSlide] = React.useState(0);
+	const	[isVisible, set_isVisible] = React.useState(true);
+
+	function	onTryToClose(): void {
+		if (onClose) {
+			onClose();
+		} else {
+			set_isVisible(false);
+		}
+	}
+
+	function	renderPreviousChevron(): ReactElement {
+		if (currentSlide === 0)
+			return (<IconChevron className={'w-4 h-4 opacity-50 cursor-not-allowed'} />);
+		return (
+			<IconChevron
+				className={'w-4 h-4 cursor-pointer'}
+				onClick={(): void => set_currentSlide(currentSlide - 1)} />
+		);
+	}
+
+	function	renderNextChevron(): ReactElement {
+		if (currentSlide === (children as ReactElement[]).length - 1)
+			return (<IconChevron className={'w-4 h-4 opacity-50 rotate-180 cursor-not-allowed'} />);
+		return (
+			<IconChevron
+				className={'w-4 h-4 rotate-180 cursor-pointer'}
+				onClick={(): void => set_currentSlide(currentSlide + 1)} />
+		);
+	}
+
+	return (
+		<div className='relative'>
+			{canClose ? <button onClick={onTryToClose} className={'absolute top-4 right-4 z-50'}>
+				<IconCross className={'w-6 h-6 cursor-pointer'} />
+			</button> : null}
```

Comment:
Only displayed if the banner should be closable. This close the whole banner system.